src/bindings/Vscode.mli

```diff
@@ -211,9 +211,9 @@ module Workspace : sig
 
   type cancellationToken = { isCancellationRequested : bool }
 
-  val name : unit -> string option
+  val name : string option
 
-  val workspaceFolders : unit -> Folder.t array
+  val workspaceFolders : Folder.t array option
```

Comment:
Why did you change the type of this function?

Reply:
Hmm good question. I guess mainly to avoid exposing a function which wraps it and returns an empty array if `undefined` or `None`; but lets the caller decides how to handle the `None` case. I can totally revert that if you prefer the previous version :)